ckanext/custom_schema/tests/test_helpers.py

```diff
@@ -0,0 +1,75 @@
+import ckan.lib.search
+from ckantoolkit.tests import helpers, factories
+
+from ckanext.custom_schema.helpers import (
+    get_group_list,
+    get_selected_group
+)
+
+
+class TestGetGroupList(object):
+    @classmethod
+    def setup(self):
+        helpers.reset_db()
+        ckan.lib.search.clear_all()
+
+    @classmethod
+    def teardown(self):
+        helpers.reset_db()
+        ckan.lib.search.clear_all()
+
+    def test_group_list_no_results(self):
+        assert get_group_list() == []
+
+    def test_group_list_with_results(self):
+        group1 = factories.Group(name='water')
+        group2 = factories.Group(name='transportation')
+        group3 = factories.Group(name='economy')
+        group_from_helpers = get_group_list()
+
+        assert group_from_helpers[0]['name'] == 'economy'
+        assert group_from_helpers[1]['name'] == 'transportation'
+        assert group_from_helpers[2]['name'] == 'water'
+
+
+class TestGetSelectedGroup(object):
+    @classmethod
+    def setup(self):
+        helpers.reset_db()
+        ckan.lib.search.clear_all()
+
+    @classmethod
+    def teardown(self):
```

Comment:
It is possible to extract setup and teardown to the base class.

Reply:
We can definitely do that. However, in favor of get the fix released sooner we can hold off on doing in another PR.